File: redsheet-portal/src/app/database-models/v-stakeholder-type.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vStakeholderType {
    constructor(vStakeholderType?: vStakeholderType )  {
        let properties = [ "actionRequried", "className", "stakeholderTypeId", "stakeholderTypeName",];
        for (let property of properties)
            if (vStakeholderType && vStakeholderType[property])
                this[property] = vStakeholderType[property];
    }
 
    @required()
    @maxLength(200)
	actionRequried : string =   undefined;
 
    @required()
    @maxLength(200)
	className : string =   undefined;
 
	stakeholderTypeId : number =   0 ;
 
    @required()
    @maxLength(200)
	stakeholderTypeName : string =   undefined;


}

File: RedSheet.DbEntities/Models/vStakeholderType.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vStakeholderTypes", Schema = "dbo")]
	[Serializable]
	public partial class vStakeholderType  
	{
 
	[Required] 
	[MaxLength(200)]
	public string ActionRequried { get; set; } 
 
	[Required] 
	[MaxLength(200)]
	public string ClassName { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int StakeholderTypeId { get; set; } 
 
	[Required] 
	[MaxLength(200)]
	public string StakeholderTypeName { get; set; } 

	public vStakeholderType()
	{
    }

    }
}

